package.json

```diff
@@ -56,7 +56,9 @@
     "tmp": "^0.1.0",
     "ts-jest": "^24.0.2",
     "ts-node": "^8.3.0",
+    "ts-transform-import-path-rewrite": "^0.2.1",
     "tsconfig-paths": "^3.8.0",
+    "ttypescript": "^1.5.7",
```

Comment:
I hope we'll get rid of it when typescript starts supporting plugins via config.

Reply:
This is not happening anytime soon unfortunately :(